LDAR_Sim/src/parameters/genric_parameters.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+
+
+class GenericParameters:
+    def __init__(self, parameters: dict[str, Any]) -> None:
+        for key, setting in parameters.items():
+            setattr(self, f"_{key}", setting)
```

Comment:
Here are you talking about this f"_{key}" string? IN this case I actually think it will make the code much more readable to avoid having this as a constant and it's not really used in multiple places.